services/index/internal/adapter/controller/http/node_handler.go

```diff
@@ -168,32 +168,34 @@ func (handler *nodeHandler) Validate(c *gin.Context) {
 
 	if err := c.ShouldBindJSON(&node); err != nil {
 		restErr := resterr.NewBadRequestError("Invalid JSON body.")
-		c.JSON(restErr.Status(), restErr)
+		c.JSON(restErr.StatusCode(), restErr)
 		return
 	}
 
 	linkedSchemas, ok := getLinkedSchemas(node)
 	if !ok {
-		c.JSON(http.StatusBadRequest, gin.H{
-			"status":          "validation_failed",
+		c.JSON(http.StatusOK, gin.H{
 			"failure_reasons": []string{"Could not read linked_schemas"},
```

Comment:
Also please change the wording of this `failure_reasons` message to: "Could not find the linked_schemas property."

Reply:
@TingSyuanWang Actually it can be made clearer: "The submitted profile does not contain the linked_schemas property."